src/SayMoreTests/model/ProjectTests.cs

```diff
@@ -347,6 +347,20 @@ public void SetContributorsListToSession_ContributorsInMetaAndSession_AllContrib
 			Assert.AreEqual(4, sessionXml.SelectSingleNode("//participants").InnerText.Split(';').Length, "Expecting four participants (with roles): Iskender Demirel, Lahdo Agirman, Greg Trihus, Eliyo Acar");
 		}
 
+		[Test]
+		[RequiresSTA]
+		public void SetContributorsListAndMigrateAccessProtocolToSession_MigrateAccessProtocol_NewAccessProtocolNodeCreated()
```

Comment:
The fact that this test really only tests the "MigrateAccessProtocol" portion of the method is further evidence to suggest that it should be two different methods, rather than one.